Assets/Scripts/dump_BulletUI.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class dump_BulletUI : MonoBehaviour
```

Comment:
classes start with capital letters. also decide between `DumpBulletUi` or `dump_bullet_ui`

Reply:
the script will be removed when a more permanent UI is in place.